dash/testing/browser.py

```diff
@@ -363,3 +371,7 @@ def server_url(self, value):
         """
         self._url = value
         self.wait_for_page()
+
+    @property
+    def download_path(self):
```

Comment:
test developer can use this to sandbox the download file, as it will be different for the next test execution. In old way, normally it also comes with specific folder clean up as test setup.